From 4971085a5504b63de35abe57265e470934f8382f Mon Sep 17 00:00:00 2001
From: Abhishek Srivastava <cabsriv@codeaurora.org>
Date: Tue, 11 Dec 2018 15:30:46 +0530
Subject: [PATCH 2/2] wifi: Increase kMaxStopCompleteWaitMs to 250 msec.

The current timeout value of 100msec for kMaxStopCompleteWaitMs
is not sufficient to stop the Wi-Fi HAL. QC Wi-Fi HAL also needs
to stop the event thread by sending an exit msg as part of stop WiFi HAL.
This stop operations are not completed with in the stipulated 100msec.

Hence , set this timeout value to 250msec.

Change-Id: I84c203a46793d7bf02986693fac403892c7bd48f
CRs-Fixed: 2357953
---
 wifi/1.2/default/wifi_legacy_hal.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/wifi/1.2/default/wifi_legacy_hal.cpp b/wifi/1.2/default/wifi_legacy_hal.cpp
index eaddc4e2..6b7c27da 100644
--- a/wifi/1.2/default/wifi_legacy_hal.cpp
+++ b/wifi/1.2/default/wifi_legacy_hal.cpp
@@ -34,7 +34,7 @@ static constexpr uint32_t kMaxGscanFrequenciesForBand = 64;
 static constexpr uint32_t kLinkLayerStatsDataMpduSizeThreshold = 128;
 static constexpr uint32_t kMaxWakeReasonStatsArraySize = 32;
 static constexpr uint32_t kMaxRingBuffers = 10;
-static constexpr uint32_t kMaxStopCompleteWaitMs = 100;
+static constexpr uint32_t kMaxStopCompleteWaitMs = 250;
 
 // Helper function to create a non-const char* for legacy Hal API's.
 std::vector<char> makeCharVec(const std::string& str) {
-- 
2.17.1

